### DISCUSSION TITLE
Raspotify on OSMC (Pi v3b) no longer working?

### DISCUSSION THREAD

**Discussion by ErikZandboer** [dtcooper/raspotify#527] [Q&A] [answered]
Hi,

Recently my OSMC crashed due to a broken SD card. I just reinstalled the latest OSMC version on my Rpi v3b, but it seems I can no longer install raspotify? I get an error around dependencies:

The following packages have unmet dependencies:
 raspotify : Depends: libc6 (>= 2.31) but 2.28-110.4 is to be installed
             Depends: libasound2 (>= 1.2.4) but 1.1.8-1 is to be installed
             Depends: alsa-utils (>= 1.2.4) but 1.1.8-2 is to be installed
             Depends: libpulse0 (>= 14.2) but 12.2-4+deb10u1 is to be installed
             Depends: systemd (>= 247.3) but 241-7~deb10u8 is to be installed
             Depends: init-system-helpers (>= 1.60) but 1.56+nmu1 is to be installed

Problem is, all the packages mentioned are on their latest version (the versions required seem unavailable). Could I solve this by installing an older version of raspotify? (and if yes: how? I couldn't get "apt-get install raspotify=0.31.10...." working).

Thanks!

**Comment by JasonLG1979** [comment, marked as answer]
I'm not sure what version of Debian OSMC is based on but I'd guess 10 which is not compatible with the latest version of Raspotify. Your only option is to remove the repo and install an old version. You'll find older versions  at the release page.
